modules/custom/openy_activity_finder/js/programs_search.js

```diff
@@ -394,45 +491,71 @@
           component.table = data.table;
           component.count = data.count;
           component.pages[component.current_page + 1] = data.pager;
+          component.pager_info = data.pager_info;
+          component.no_results = data.count > 0 ? '0' : '1';
+          component.sort = data.sort;
+
+          router.push({ query: {
+            locations: cleanLocations.join(','),
+            categories: cleanCategories.join(','),
+            ages: cleanAges.join(','),
+            days: cleanDays.join(','),
+            keywords: component.keywords,
+            page: component.page,
+            no_results: component.no_results,
+            sort: component.sort,
+          }});
+        }).done(function() {
+          // We need to wait in order to affect the DOM after the tiles have been injected.
```

Comment:
comment makes no sense. We are not waiting here, right?

Reply:
fixed